perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -84,9 +84,11 @@ LIST_CONFIG_FILE="config-hns.json"
 run_load_test_and_fetch_metrics "$GCSFUSE_FIO_FLAGS" "$BUCKET_NAME" "$SPREADSHEET_ID"
 run_ls_benchmark "$GCSFUSE_LS_FLAGS" "$SPREADSHEET_ID" "$LIST_CONFIG_FILE"
 
-#Running the rename benchmark script.
+# Running the rename benchmark script.
 cd "./hns_rename_folders_metrics"
-./run_rename_benchmark.sh $UPLOAD_FLAGS
+# Rename perf tests are only performed daily on HNS bucket by default.Thus , creating log file with hns suffix only.
+LOG_FILE_RENAME_TESTS=${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs-rename-hns.txt
```

Comment:
We have to add this prefix [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/continuous_test/gcp_ubuntu/continuous.cfg#L15).